prow/config.yaml

```diff
@@ -409,6 +409,20 @@ presets:
       - key: secret
         path: id_rsa
         mode: 0600
+- labels:
+    preset-enable-netrc: "true"
```

Comment:
We should not generate jobs with presets, please add the secrets and mounts directly to each pod.

Reply:
I will add an option and support for this (e.g. `--expand-presets`). This functionality should be user-defined if they want to use presets or expand them. This tool is used for Istio GOB (and potentially other places in the future) - places where job *readability* is be preserved and the convenience of a *preset* is justified.

Reply:
Will add this in a follow-up:
https://github.com/istio/test-infra/issues/2096